tests/ext/test_application.py

```diff
@@ -277,22 +278,35 @@ async def test_shutdown(self, one_time_bot, monkeypatch, updater):
         def after_bot_shutdown(*args, **kwargs):
             self.test_flag.add("bot")
 
+        def after_shutdown_update_processor(*args, **kwargs):
+            self.test_flag.add("update_processor")
+
         def after_updater_shutdown(*args, **kwargs):
             self.test_flag.add("updater")
 
+        update_processor = SimpleUpdateProcessor(1)
         monkeypatch.setattr(Bot, "shutdown", call_after(Bot.shutdown, after_bot_shutdown))
+        monkeypatch.setattr(
+            SimpleUpdateProcessor,
+            "initialize",
+            call_after(SimpleUpdateProcessor.initialize, after_shutdown_update_processor),
```

Comment:
```suggestion
            "shutdown",
            call_after(SimpleUpdateProcessor.shutdown, after_shutdown_update_processor),
```
? how did the tests pass

Reply:
seems scary 😬